agent/profile

```diff
@@ -6,6 +6,13 @@ function _pbench_pathins() {
     fi
 }
 
+_pbench_pathins "/usr/local/sbin"
+_pbench_pathins "/usr/local/bin"
```

Comment:
I don't these these two should be included.  Does the pbench-agent RPM install anything in these directories?  If not, then we should not include them.  It is up to the administrator of the system to add them to the path explicitly.

Reply:
Yes the pbench-agent rpm doesnt seem to install anything in these 2 directories, I will remove them